modules/mssql/mssql_test.go

```diff
@@ -128,3 +128,125 @@ func TestMSSQLServerWithInvalidPassword(t *testing.T) {
 	testcontainers.CleanupContainer(t, ctr)
 	require.NoError(t, err)
 }
+
+// tests that a container can be created with a DDL script
+func TestMSSQLServerWithScriptsDDL(t *testing.T) {
+	const password = "MyCustom@Passw0rd"
+	seedFile := filepath.Join("testdata", "seed.sql")
+
+	tests := []struct {
```

Comment:
suggestion: use t.Run instead of test table as that allows specific tests to be run from editors and also reduces the amount of code.

```golang
t.Run("password-then-scripts", func(t *testing.T) {
      testWithScripts(t, mssql.WithScripts(seedFile), mssql.WithPassword(password))
}

t.Run("scripts-then-password", func(t *testing.T) {
      testWithScripts(t, mssql.WithScripts(seedFile), mssql.WithPassword(password))
}

...

// testWithScripts tests the behaviour of WithScripts
func testScripts(t *testing.T, options ...testcontainer.ContainerCustomiser) {
     t.Helper()

    // Content for t.Run from the for loop goes here.
}
```